### PR TITLE
ltp: Add ltp ipc tests using Linaro test-definitions

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Signed-off-by: Lakshmipathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
Job: https://lava.collabora.co.uk/scheduler/job/2635956

File: templates/ltp/ltp.jinja2 [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-          steps:
-          - cd /opt/ltp/ && ./runltp -p -f syscalls -s ^open02 -l /tmp/results.log
-          - cat /tmp/results.log
-          - lava-test-case check-result --shell grep 'PASS' /tmp/results.log
-      from: inline
-      name: {{ plan }}
-      path: inline/{{ plan }}.yaml
+       - repository: https://github.com/kernelci/test-definitions
+         from: git
+         revision: master
```
Comment (gctucker) [inline templates/ltp/ltp.jinja2:8]: Please use `kernelci.org` branch.

**Review by gctucker — COMMENTED** [review]
Please see the comment about the test-definitions branch name, rename the test plan to 'ltp-ipc' so we can have other variants to run other parts of LTP.

File: templates/ltp/ltp.jinja2 (hunk 1 above)
Comment (Lakshmipathi) [reply templates/ltp/ltp.jinja2]: ok, updated the patch.

File: templates/ltp/ltp.jinja2 [hunk 2]
```diff
@@ -1,21 +1,14 @@
...
-          - cat /tmp/results.log
-          - lava-test-case check-result --shell grep 'PASS' /tmp/results.log
-      from: inline
-      name: {{ plan }}
-      path: inline/{{ plan }}.yaml
+       - repository: https://github.com/kernelci/test-definitions
+         from: git
+         revision: kernelci.org
+         path: automated/linux/ltp/ltp.yaml
+         name: ltp-ipc
```
Comment (gctucker) [inline templates/ltp/ltp.jinja2:10]: This should be passed as an argument, see `v4l2-compliance` and `igt`.

File: templates/ltp/ltp.jinja2 [hunk 3]
```diff
@@ -1,21 +1,14 @@
 - test:
+    name: ltp-ipc
```
Comment (gctucker) [inline templates/ltp/ltp.jinja2:2]: This should be passed as a template argument.

**Review by gctucker — CHANGES_REQUESTED** [review]
Please rename to `ltp-ipc` in `test-configs.yaml` and use the same mechanism as for v4l2-compliance and igt to define variants.  The results should appear as an `ltp-ipc` test plan, so we can run other LTP-based test plans separately but using the same `ltp.jinja2` template.

**Comment by Lakshmipathi** [comment]
LAVA job with updated changes: https://lava.collabora.co.uk/scheduler/job/2653221

**Comment by gctucker** [comment]
@Lakshmipathi Thanks - can you please rebase this PR? It appears to be conflicting now.

**Comment by Lakshmipathi** [comment]
ok, now the patch rebased on master.

**Comment by gctucker** [comment]
Great, first results can now be seen on staging:
https://staging.kernelci.org/test/plan/id/5f719d53813243286aa69d3c/

**Review by gctucker — APPROVED** [review]

**Pull request merged**